src/js/ImportController.js

```diff
@@ -164,7 +155,8 @@ goog.require('ga_wmts_service');
         // non-SwissProjected test urls
         'http://wms.ga.admin.ch/1GE',
         'http://wms.ga.admin.ch/LG_DE_Geologie_und_Tektonik/wms',
-        'http://discomap.eea.europa.eu/arcgis/services/Land/CLC2000_Dyna_WM/MapServer/WMSServer',
+        'http://bio.discomap.eea.europa.eu/arcgis/services/Ecosystem/Ecosystems/MapServer/WMSServer?request=GetCapabilities',
```

Comment:
no need to write  "?request=GetCapabilities"

Reply:
agree